scipy/stats/_continuous_distns.py

```diff
@@ -3591,8 +3592,65 @@ class levy_stable_gen(rv_continuous):
 
     Notes
     -----
-    Levy-stable distribution (only random variates available -- ignore other
-    docs)
+    The distribution for `levy_stable` has characteristic function:
+
+    .. math::
+            
+        \varphi(t; \alpha, \beta, c, \mu) = e^{ it\mu -|ct|^{\alpha }(1-i\beta \operatorname{sign}(t)\Phi(\alpha ,t )) }
+
+    where:
+              
+    .. math::
+        
+        \Phi = \begin{cases}
+                \tan \left({\frac {\pi \alpha }{2}}\right)&\alpha \neq 1\\
+                -{\frac {2}{\pi }}\log |t|&\alpha =1
+                \end{cases}
+           
+    The probability density function for `levy_stable` is:
+        
+    .. math::
+    
+        f(x) = \frac{1}{2\pi}\int_{-\infty}^\infty \varphi(t)e^{-ixt}\,dt
+        
+    where :math:`-\infty < t < \infty`. This integral does not have a known closed form.
+    
+    For evaluation of pdf we use either Zolotarev :math:`S_0` parameterization with integration, 
+    direct integration of standard parameterization of characteristic function or FFT of 
+    characteristic function. FFT is used if number of points is greater than 
+    ``levy_stable.pdf_fft_min_points_threshold`` (defaults to 100) otherwise we use one of the 
+    other methods. Setting the threshold to ``None`` will disable FFT. The default method is 
+    Zolotarev's but can be changed by setting ``levy_stable.pdf_default_method`` to any string 
+    other than 'zolotarev'. To increase accuracy of FFT calculation one can specify 
+    ``levy_stable.pdf_fft_grid_spacing`` (defaults to 0.01) and ``pdf_fft_n_points_two_power`` 
+    (defaults to a value that covers the input range * 4). Setting ``pdf_fft_n_points_two_power`` 
+    to 16 should be sufficiently accurate in most cases at the expense of CPU time.
+    
+    This implementation of Zolatarev is unstable for some values where alpha = 1 and beta != 0 or for 
+    very small values of alpha. In this case the quadrature method is recommended.
```

Comment:
This warning doesn't quite match what you said in https://github.com/scipy/scipy/pull/8766#issuecomment-396062917. Can this be more precise (e.g. `in particular when |alpha-1| < 0.005; alpha < 0.2 or alpha=1 and beta is very small `)

Reply:
also, would be good to use:

    .. warning::

        text about alpha ranges

so this actually gets noticed by users.

Reply:
Just regarding my previous comment, that only applies to the output from Nolan's stablec and may or may not apply to our methods. Since his executable is known to be unstable for alpha < 0.2 we don't extract data for that range. Specifically, we now extract densities for alpha = .25, .5, .75, 1, 1.25, 1.5 and 2, beta = -1, -.5, 0, .5, 1 and x = -10..10 (step 1) and check each method against subsets of these ranges up to specific decimal precision. Hope that sounds reasonable.

Reply:
Okay, thanks for the clarification